src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
+      'line-height': 'inherit',
+      padding: '0px',
+      'margin-top': '12px',
```

Comment:
Here's what it looks like with `margin: 6px 0 0` `padding: 0 0 6px`

![screen shot 2017-04-09 at 2 51 50 pm](https://cloud.githubusercontent.com/assets/3895824/24840058/1574bcca-1d34-11e7-8888-341ea4ec554b.png)

the over flow still shows up a bit under the label